### ISSUE TITLE
On what Recipes should be included.

### ISSUE THREAD

**Issue by LukeSmithxyz** [LukeSmithxyz/based.cooking#377]
I have basically been accepting everything, including things I would not included under the umbrella of "based."
Now that we have "enough" recipes to make it a usable site, I want to be more restrictive in what is accepted.
Feel free to say your views on it.

On drinks and smoothies, there have been a couple smoothies or things like them and I frankly feel like they're a waste. That's not a real recipe. A "blueberry and banana smoothie" has its recipe in its name, more or less. Possibly some drinks might be allowed, but I am open to removing them altogether. I'm not sure if cocktails are really appropriate either, I mean I'm not necessarily against them, but I think a case can be made that it's not really relevant.

The only thing I have specifically not accepted is meat-substitute cringe which is an unambiguous case.
I will say that my own biases are to mostly not accept desserts.
Those of you who know me know that I am very anti-sugar and [view it as the cause of all ills](https://videos.lukesmith.xyz/videos/watch/733b53cb-b4df-4cb1-9615-045fbd6789dd), but I suppose there could be based dishes with it.

## New potential submission rules

These are to slim down submissions. Offer more suggestions.

- Images **must** be included and must've been taken by the submitter. This encourages people to check their recipes one more time and not just submit a million things, but focus on what matters. I might say that images must be less than 150K.
- Recipes should be Lindy (Lindy -> Based). That is, recipes should be longstanding folk recipes, not modern abominations. Most of what we have is fine, but some are border line.
- People need to spend time to link redundancies. If a recipe is already present that shows you how to make boil pasta or some other ingredient, it should be linked in the recipe page.

## On tagging

https://github.com/LukeSmithxyz/based.cooking/pull/240 Is a nice PR which I haven't merged that adds fasting information.
I will not be so cringe as to accept "diet" tags, but I think this is an acceptable and helpful thing.
The issue is whether new submitters can keep up with it.

I don't know if other people consider it a waste to have this, I might, just because you sort of know by looking at the ingredients what is acceptable, but I suppose it would be nice to have a tag or it.

**Comment by hallzy** [comment]
I agree for most of this.

Most drinks as you point out probably don't need a recipe. I'd even go more general and say that if the name of the recipe fully explains it (like blueberry banana smoothie as an example), or something like a 'fruit salad' which is just combining whatever raw fruits you want, that we don't need a recipe for it.

I would personally like desserts, but you've stated from the beginning that you didn't want sugary things so I am okay with this.

I think requiring images would be good. I always find it useful to be able to see the end product and right now a lot of recipes are missing pictures.

**Comment by yiusa** [comment]
I agree with most of what you have written up here except the part about images.
I know that having images makes the recipes a lot more clear to understand, and also makes them more presentable but most of my recipes are writen down in notebooks ive made through the years and when i travel.  It would be quite unfeasable for me to remake every recipe in my books just to share it. 

I completely agree that overly sugary recipes, and things like the smoothie or alcoholic recipe you mention don't really have to be on the website since i don't think that that is the direction you'd like to take it. 

On the topic of meat substitute recipes, i have collected a couple extremely delicious tofu recipes when i went to china, i'd consider these ok since they're traditional recipes and the tofu isn't used as a meat-substitute, its just another ingredient, but I'd like to hear what you guys think about these.

**Comment by RyanWilson7** [comment]
Good grammar and spelling should be a submission rule, if it isn't already imo. Also, how would you define a modern abomination?

**Comment by taco-c** [comment]
Regarding drinks, I think they have a place on the site, but under the same requirements as the "normal" recipes, i.e. traditional and folksy is ok (like the Glühwein and maybe some teas, etc.).

Also I would personally like having basic/generic "recipes" for e.g. preservation methods and the like.

Other than that, I agree with quality over quantity.

**Comment by LukeSmithxyz** [comment]
> Good grammar and spelling should be a submission rule, if it isn't already imo. Also, how would you define a modern abomination?

Sure it should be a rule, but I'm not going to be able to check it all. If I tell someone to fix an error, it'll will multiply the time I have to worry about the PR by more than 2.

**Comment by LukeSmithxyz** [comment]
> On the topic of meat substitute recipes, i have collected a couple extremely delicious tofu recipes when i went to china, i'd consider these ok since they're traditional recipes and the tofu isn't used as a meat-substitute, its just another ingredient, but I'd like to hear what you guys think about these.

I am totally 100% fine with recipes with bean curd or anything else that might be used as a meat-substitute if it is indeed part of a traditional dish.

**Comment by m-chrzan** [comment]
>On drinks and smoothies, there have been a couple smoothies or things like them and I frankly feel like they're a waste. 

Totally agree on smoothies. Maybe a single "this is the general method for making smoothies" page with a few different ingredients list suggestions could work, but yeah, ten different smoothies where ingredients are the title is spam.

>Images must be included and must've been taken by the submitter.

I think a major problem with requiring images is that people are just not good at taking pictures (myself included), especially appealing pictures of food. The site would get filled with low contrast pictures of mush.

>People need to spend time to link redundancies. If a recipe is already present that shows you how to make boil pasta or some other ingredient, it should be linked in the recipe page.

On the one hand, this sounds nice and DRY. On the other hand, even the most basic recipes often have variations and people have opinions on them. A recipe author might prefer a slightly different version of tomato sauce that his grandma always made rather than the one that's posted as a recipe here.

Requiring this sort of modularization also increases complexity - recipes become hyperlinked graphs instead of just a linear text.

>Those of you who know me know that I am very anti-sugar and view it as the cause of all ills, but I suppose there could be based dishes with it.

I generally agree on anti-sugariness. I think the one case for desserts is similar to alcohol - in social settings.

**Comment by notthewave** [comment]
It might be a good idea to request that ingredients aren’t measured exactly, so people can – to a certain extent – rely on their own judgement (wich is something you need if you want to cook without a recipe)

e.g. 
a hand full of ...
five tablespoons ...
two teaspoons ...

This way imperial/metric confusion is not much of a concern anymore, and it’s easier to remember as well. (recipe independence lul)

I do realize this is not always ideal.

**Comment by hallzy** [comment]
I'd argue that exact measurements can and are important in many cases (definitely not always). Cooking is literally chemistry for normal people.

Also a tbsp and tsp are imperial measurements. 1 tsp is roughly 5 mL.

**Comment by taco-c** [comment]
> It might be a good idea to request that ingredients aren’t measured exactly, so people can – to a certain extent – rely on their own judgement (wich is something you need if you want to cook without a recipe)

I like the idea, but i don't think it's enforceable.
